openassessment/fileupload/backends/gcs.py

```diff
@@ -0,0 +1,73 @@
+"""GCS Bucket File Upload Backend."""
+
+from google.cloud import storage
+import logging
+
+from ..exceptions import FileUploadInternalError
+from .base import BaseBackend
+
+log = logging.getLogger("openassessment.fileupload.api")  # pylint: disable=invalid-name
+
+
+class GCSBackend(BaseBackend):
+    """ GCS Bucket File Upload Backend. """
+
+    def get_upload_url(self, key, content_type):
+        """Get a signed URL for uploading a file to GCS"""
+        bucket_name, key_name = self._retrieve_parameters(key)
+        try:
+            client = _connect_to_gcs()
+            bucket = client.bucket(bucket_name)
+            blob = bucket.blob(key_name)
+            url = blob.generate_signed_url(
+                version="v4",
+                expiration=self.UPLOAD_URL_TIMEOUT,
```

Comment:
I take the current default in seconds is 3600s (defined here ` openassessment/fileupload/backends/base.py`). I'm thinking if this has any implications, probably not.

Reply:
files are small for ORA, these defaults are very old for slow internet access and they are more than enough